aixcalibuha/sensitivity_analysis/fast.py

```diff
@@ -0,0 +1,81 @@
+"""
+Adds the FASTAnalyzer to the available
+classes of sensitivity analysis.
+"""
+from SALib.sample import fast_sampler as fast
+from SALib.analyze import fast as analyze_fast
+import numpy as np
+from aixcalibuha.sensitivity_analysis import SenAnalyzer
+from aixcalibuha import CalibrationClass
+import warnings
+
+
+class FASTAnalyzer(SenAnalyzer):
+    """
+    Additional arguments:
```

Comment:
A bit more verbose, i.e. "FAST method from SALib (with link to their doc)." Implemented in v.1.0.0, and what it does. Same for PAWN

Reply:
I also added it for the other old methods.